guides/source/routing.md

```diff
@@ -1134,10 +1134,17 @@ For example, here's a small section of the `rake routes` output for a RESTful ro
 edit_user GET    /users/:id/edit(.:format) users#edit
 ```
 
-You may restrict the listing to the routes that map to a particular controller setting the `CONTROLLER` environment variable:
+You may restrict the listing to the routes that map to a particular controller using the `--controller` option — or its `-c` shorthand:
+```bash
+$ bin/rake routes --controller users
+$ bin/rake routes -c users
+$ bin/rake routes -c Users
+$ bin/rake routes -c UsersController
+```
 
+You can grep your routes with `-g`, which filters routes by `name`, `verb` or `path`.
 ```bash
-$ CONTROLLER=users bin/rake routes
+$ bin/rake routes -g users
```

Comment:
I think it would make more sense to introduce the general searching first and then mention controller filtering. How about:

> You can search through your routes with the --grep option (-g for short). This outputs any routes that partially match the URL helper method name, the HTTP verb, or the URL path.
>
> ```bash
> $ bin/rake routes --grep new_comment
> $ bin/rake routes -g POST
> $ bin/rake routes -g admin
> ```
>
> If you only want to see the routes that map to a specific controller, there's the --controller option (-c for short).
>
> ```bash
> $ bin/rake routes --controller users
> $ bin/rake routes --controller admin/users
> $ bin/rake routes -c Comments
> $ bin/rake routes -c Articles::CommentsController
> ```

By the way, does the controller option support module namespaces like the last example?